packages/components/pie-toggle-switch/src/toggle-switch.scss

```diff
@@ -134,6 +137,12 @@
 }
 
 :host([dir='rtl']) {
+    .c-toggleSwitch-control {
+        position: absolute;
```

Comment:
i think we can remove position: absolute here since it was declared above

happy to include it in my upcoming PR if that's fine

Reply:
Will fix in a follow up if that's ok.